src/pudl/helpers.py

```diff
@@ -1808,12 +1810,19 @@ def get_dagster_execution_config(num_workers: int = 0):
         num_workers: The number of workers to use for the dagster execution config.
             If 0, then the dagster execution config will not include a multiprocess
             executor.
+        tag_concurrency_limits: A set of limits that are applied to steps with
+            particular tags. This is helpful for applying concurrency limits to
+            highly concurrent and memory intensive portions of the ETL like CEMS.
+
+            Dagster description: If a value is set, the limit is applied to only
+            that key-value pair. If no value is set, the limit is applied across
+            all values of that key. If the value is set to a dict with
+            `applyLimitPerUniqueValue: true`, the limit will apply to the number
+            of unique values for that key. Note that these limits are per run, not global.
 
     Returns:
         A dagster execution config.
     """
-    if not num_workers:
-        return {}
```

Comment:
Just using the default values of the function is simpler now that we have multiple execution config arguments.

Reply:
I think maybe I would benefit from a walkthrough on the config system. I don't really understand how these settings are flowing into the run, when they get used vs. not.

Reply:
Does this change mean that the asset groups would have a concurrency of 2 for cems now too?

Reply:
No :/ I'm not sure of a way to configure execution config for asset groups. Basically, you need to construct a job from a collection of assets then specify the execution configuration for that job.